### ISSUE TITLE
search redirects to broken captcha

### ISSUE THREAD

**Issue by Granitosaurus** [QuackMax/rugle#1]
Clicking search takes you to google translate page with unfunctioning captcha: 
![image](https://user-images.githubusercontent.com/5476164/171114456-22323f73-0d0b-43be-b09d-97953f1644dc.png)

**Comment by QuackMax** [comment]
Yes, I've raised the issue with Google. It seems to be a problem on their side, their site key needs updating for reCAPTCHA to work properly.  
  
You can get around that by using a VPN, a different network, or trying again in ~5-10 minutes. Sometimes it'll break when going through mobile data but not through wifi, or vice-versa. 
  
If and when Google fixes it, it'll likely lead to a functional reCAPTCHA you can solve to access the search results. Sorry about that, I'm not aware of a way to resolve that myself in the immediate. 
    
If you're curious, the specific part that breaks is the request for Google Translate to translate the Google Search results page (that is, the search terms are properly translated, the resultant URL is well-formed, and future attempts with the exact same terms and languages will likely work once Google chills out)   
   
I'll leave the issue open for now, in case a workaround can be found

**Comment by Granitosaurus** [comment]
> If and when Google fixes it

I admire your optimism :)

**Comment by QuackMax** [comment]
I'd imagine a broken captcha leads to user rejection and lost revenue somewhere down the line, since if it's broken for Rugle it's likely broken  in general   =o)  (me with my clown nose on, prepping the next step of the costume)
